modules/manage/pages/audit-logging.adoc

```diff
@@ -0,0 +1,66 @@
+
+By default, any management and authentication actions performed on the cluster yield messages written to the audit log topic that are retained for seven days. Interactions with all topics by all principals are audited. Actions performed using the Kafka API and Admin API are all audited, as are actions performed directly through rpk.
+
+Messages recorded to the audit log topic comply with the https://schema.ocsf.io/[open cybersecurity schema framework]. Any number of analytics frameworks, such as Splunk or Sumo Logic, can receive and process these messages. Using an open standard ensures Redpanda's audit logs coexist with those produced by other IT assets, powering holistic monitoring and analysis of your assets.
+
+== Audit Log Configuration Options
+
+Redpanda's audit logging mechanism supports several options to control the volume and availability of audit records. Configuration is applied at the cluster level using the standard https://docs.redpanda.com/current/manage/cluster-maintenance/cluster-property-configuration/[cluster configuration mechanism]. The cluster configuration options exclusive to audit logging include the following:
+
+* `audit_enabled` : Boolean value to enable audit logging. When you set this to true, Redpanda checks for an existing topic named `__audit_log`. If none is found, Redpanda automatically creates one for you. (default: false)
+* `audit_log_num_partitions` : Integer value defining the number of partitions used by a newly created audit topic. This configuration applies only to the audit log topic and may be different from the cluster or other topic configurations. This cannot be altered for existing audit log topics. (default: 12)
+* `audit_log_replication_factor` : Integer value defining the replication factor for a newly created audit log topic. This configuration applies only to the audit log topic and may be different from the cluster or other topic configurations. This cannot be altered for existing audit log topics. (default: 3)
+* `audit_client_max_buffer_size` : Integer value defining the number of bytes allocated by the internal audit client for audit messages. When changing this, you must disable audit logging and then re-enable it for the change to take effect. Consider increasing this if your system generates a very large number of audit records in a short amount of time. (default: 16MiB)
+* `audit_queue_max_buffer_size_per_shard` : Integer value defining the maximum amount of memory used by the audit buffer in each shard. Once this size is reached, requests to log additional audit messages will return a non-retryable error. You must restart the cluster when changing this value. (default: 1MiB)
+* `audit_enabled_event_types` : List of strings in JSON style identifying the event types to include in the audit log. This may include any of the following - `management, produce, consume, describe, heartbeat, authenticate`. (default: `'["management","authenticate"]'`)
```

Comment:
```suggestion
* `audit_enabled_event_types` : List of strings in JSON style identifying the event types to include in the audit log. This may include any of the following - `management, produce, consume, describe, heartbeat, authenticate`. Default: `'["management","authenticate"]'`).
```